### ISSUE TITLE
Next/prev buttons - accessibility update

### ISSUE THREAD

**Issue by jvine** [ProjectMirador/mirador#2019]
Two changes required to `mirador-previous-canvas-button` and `mirador-next-canvas-button`

- [ ] Move the buttons in the DOM so they fall _after_ the OSD container and zoom controls. (Tab order is conceptually: I'm on this image, I can do things to this image, I can go to the next image)
- [ ] Label the buttons `Previous item` and `Next item` (currently they have no labels)
